public/bookmarklet.html

```diff
@@ -0,0 +1,112 @@
+                Items from this page
+              </label>
+            </div>
+          </div>
+          <div class="modal-footer">
+            <button type="button" class="btn btn-link" data-dismiss="modal">Cancel</button>
+            <button type="button" class="btn btn-primary" id="collabosphere-overview-next">Next</button>
+          </div>
+        </div>
+
+        <div id="collabosphere-bookmark" class="hide collabosphere-pane">
+          <h5>Add current page as bookmark</h5>
+          <form class="form-horizontal">
+            <div class="form-group">
+              <label for="collabosphere-bookmark-url" class="col-sm-2 control-label">URL</label>
+              <div class="col-sm-10">
+                <input class="form-control" id="collabosphere-bookmark-url" disabled="disabled">
+              </div>
+            </div>
+            <div class="form-group">
+              <label for="collabosphere-bookmark-title" class="col-sm-2 control-label">Title</label>
+              <div class="col-sm-10">
+                <input class="form-control" id="collabosphere-bookmark-title" placeholder="Enter title">
+              </div>
+            </div>
+            <div class="form-group">
+              <label for="collabosphere-bookmark-description" class="col-sm-2 control-label">Description</label>
+              <div class="col-sm-10">
+                <textarea class="form-control" id="collabosphere-bookmark-description" placeholder="Enter description" rows="3"></textarea>
+              </div>
+            </div>
+            <div class="modal-footer">
+              <button type="button" class="btn btn-link collabosphere-back">Back</button>
+              <button type="button" class="btn btn-primary" id="collabosphere-bookmark-add">Add</button>
+            </div>
+          </form>
+        </div>
+
+        <div id="collabosphere-items" class="hide collabosphere-pane">
+          <h5>Select the items you'd like to add</h5>
+          <ul id="collabosphere-items-list" class="list-inline clearfix"><!-- --></ul>
+          <div class="modal-footer">
+            <button type="button" class="btn btn-link collabosphere-back">Back</button>
+            <button class="btn btn-primary" id="collabosphere-items-next" type="button">Next</button>
+          </div>
+        </div>
+
+        <div id="collabosphere-items-metadata" class="hide collabosphere-pane">
+          <h5>Add more information about the items you'd like to add</h5>
+          <ul id="collabosphere-items-metadata-container" class="list-unstyled collabosphere-items-metadata-container"><!-- --></ul>
+          <div class="modal-footer">
+            <button type="button" class="btn btn-link" id="collabosphere-items-metadata-back">Back</button>
+            <button type="button" class="btn btn-primary" id="collabosphere-items-add">Add</button>
+          </div>
+        </div>
+
+        <div id="collabosphere-done" class="hide collabosphere-pane"><!-- --></div>
```

Comment:
@nicolaasmatthijs where is this used? Or is it just a placeholder for the future?